migrations/Migration20240909061707.ts

```diff
@@ -0,0 +1,20 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20240909061707 extends Migration {
+
+  public up(): void {
+    this.addSql('create type "email_address_status_enum" as enum (\'ENABLED\', \'DISABLED\', \'REQUESTED\');');
+    this.addSql('alter table "email_address" drop column "enabled";');
+
+    this.addSql('alter table "email_address" add column "status" "email_address_status_enum" not null;');
+  }
```

Comment:
Shouldn't this also set the status on all existing rows in the table? Otherwise the `not null` constraint would be violated